python/tests/kat/t_basics.py

```diff
@@ -30,6 +36,7 @@ def config(self) -> Generator[Union[str, Tuple[Node, str]], None, None]:
     def queries(self):
         yield Query(self.url("ambassador/v0/diag/?json=true&filter=errors"), phase=2)
         yield Query(self.url("_internal/v0/ping", scheme="http", port=8877), expected=403)
+        yield Query(self.url("ambassador/v0/check_ready", scheme="http", port=8877))
```

Comment:
Shouldn't this use port 8500?

Reply:
No, it can't because that Listener only runs on localhost (127.0.0.1) thus is not available to the `kat-client` pod where all the kat requests come from.

Reply:
https://github.com/emissary-ingress/emissary/blob/master/cmd/entrypoint/healthcheck_server.go#L67

Copying from slack for context. It comes in on 8877 --> ambwatch.Ready check --> points at new Listener (127.0.0.1:8500) which is indirectly verifying two things:

1. that the new Listener is in fact working when configured with a different port
2. that diagd is able to process snapshots

Reply:
Ah right, I'm misremembering the flow through `entrypoint`. 🤦‍♂️